test/models/report_test.rb

```diff
@@ -3,7 +3,30 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  include Devise::Test::IntegrationHelpers
+  setup do
+    @report = reports(:one)
+    @alice = users(:alice)
+    @bob = users(:bob)
+    sign_in @alice
+  end
+
+  test '#editable?' do
+    report = Report.new(title: 'hoge', content: 'fuga', user: @alice)
+    assert report.editable?(@alice)
+    assert_not report.editable?(@bob)
+  end
+
+  test '#creat_on' do
+    assert_equal Date.current, @report.created_on
+  end
+
+  test '#mentions_test' do
+    not_mention = Report.new(title: 'mention_test', content: 'hoge', user: @alice, )
+    not_mention.save
```

Comment:
この書き方だとバリデーションエラーが起きて保存に失敗してもすぐに気づけません。
以下のように!付きのメソッドを使いましょう（失敗したら例外が起きてそこでテストが止まる）

```suggestion
    not_mention.save!
```

Reply:
修正しました。